lib/cf/broker/get-usage.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+set -e
+
+CLIENT_ID=abacus
+CLIENT_SECRET='s3cret'
```

Comment:
Do we need this script as part of the broker? We might move it to another helper repo.

The proper place for scripts is in `bin` dir of the project. If these scripts are used for Concourse we might place them in `etc/concourse/scripts`